server/controllers/errors/clientError.js

```diff
@@ -0,0 +1,8 @@
+const clientError = (req, res) => {
+  res.status(404).send({
+    StatusCode: '404',
```

Comment:
there is no need to repeat statusCode here, you already add it in the previous line